docs/UserGuide.md

```diff
@@ -223,19 +263,36 @@ Examples:
 *  `list` followed by `edit 1 p/12345678 animal/Bob` edits the phone number and the pet name of the 1st fosterer in the address book to 12345678 and Bob respectively.
```

Comment:
does this only work with indices shown after the 'list' command? I reviewed nicole's UG and it seems like it will work for the list shown after 'find' command also, might have to check! for reference, this is what she wrote "find Jerry or list Jerry, followed by delete 1, deletes the 1st fosterer in the resulting list", same question for view command (any command that uses index)